src/main/io/vtx_tramp.c

```diff
@@ -0,0 +1,208 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* Created by jflyper */
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <ctype.h>
+#include <string.h>
+
+#include "platform.h"
+
+#ifdef VTX_TRAMP
+
+#include "build/debug.h"
+
+#include "common/utils.h"
+
+#include "io/serial.h"
+#include "drivers/serial.h"
+#include "drivers/vtx_var.h"
+#include "io/vtx_tramp.h"
+
+static serialPort_t *trampSerialPort = NULL;
+
+static uint8_t trampCmdBuffer[16];
+
+static void trampWriteBuf(uint8_t *buf)
+{
+    serialWriteBuf(trampSerialPort, buf, 16);
+}
+
+static uint8_t trampChecksum(uint8_t *buf)
+{
+    uint8_t cksum = 0;
+
+    for (int i = 0 ; i < 14 ; i++)
```

Comment:
The first byte should not included in the checksum (see prot def). If I change the start value to 1, channels can be changed.